lib/Backend/GlobOpt.h

```diff
@@ -1008,6 +1009,7 @@ class GlobOptBlockData
     CapturedValues *                        capturedValuesCandidate;
     CapturedValues *                        capturedValues;
     BVSparse<JitArenaAllocator> *           changedSyms;
+    BVSparse<JitArenaAllocator> *           changedSymsAfterIncBailoutCandidate;
```

Comment:
Yes, moved it to be a member of GlobOpt.

---
In reply to: [72304117](https://github.com/Microsoft/ChakraCore/pull/1337#discussion_r72304117) [](ancestors = 72304117)